File: app/src/main/java/io/lerk/lrkFM/entities/FMArchiveFile.java
package io.lerk.lrkFM.entities;

import java.io.File;

/**
 * @author Lukas Fülling (lukas@k40s.net)
 */
class FMArchiveFile extends FMFile {

    /**
     * Constructor.
     *
     * @param f the file
     */
    FMArchiveFile(File f) {
        super(f);
    }

    void setDirectory(boolean b) {
        this.directory = b;
    }
    void setAbsolutePath(String path) {
        this.absolutePath = path;
    }
}

File: app/src/main/java/io/lerk/lrkFM/entities/FMArchiveFile.kt
package io.lerk.lrkFM.entities

import java.io.File

/**
 * Archive file (file inside an archive).
 *
 * @author Lukas Fülling (lukas@k40s.net)
 */
internal class FMArchiveFile(f: File) : FMFile(f) {
    var directory: Boolean? = null
}